examples/go/queryProofData/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 
 	// query proof
 	var data HeaderF
-	err = api.Client.Call(&data, "kate_queryDataProof", transactionIndex, h)
+	err = api.Client.Call(&data, "kate_queryDataProofv2", transactionIndex, h)
```

Comment:
This example won't run because `kate_queryDataProofv2` return a different type of object than 
`kate_queryDataProof`